products/installation-cpaas.yaml

```diff
@@ -59,7 +59,7 @@ products:
   3scale:
     channel: "alpha"
     installFrom: "implicit"
-    bundle: "registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata@sha256:ab721f45dd85c53f8e36acdc32424ecf97bd621ae601d9fe4c1c74cf2d6a8eea"
+    bundle: "registry.redhat.io/3scale-amp2/3scale-rhel7-operator@sha256:5063f838a8a7649626231edabf47debc3d4e36f69de37675d126b19fdbeb69c3"
```

Comment:
Just a note: the image was replaced to address the error in OVP:
```
[2021-07-16T16:16:05.641Z] Checking registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata@sha256:ab721f45dd85c53f8e36acdc32424ecf97bd621ae601d9fe4c1c74cf2d6a8eea

[2021-07-16T16:16:05.641Z] Error: Image registry-proxy.engineering.redhat.com/rh-osbs/3scale-amp2-3scale-rhel7-operator-metadata@sha256:ab721f45dd85c53f8e36acdc32424ecf97bd621ae601d9fe4c1c74cf2d6a8eea is not located in an approved Red Hat registry
```

[Info about the new image](https://catalog.redhat.com/software/containers/3scale-amp2/3scale-rhel7-operator/5d80b904bed8bd3809e0d2e3?tag=1.13.0-17.1623245035&architecture=amd64&push_date=1623774988000&container-tabs=overview)